[2024-05-09T22:03:24.768-0300] {processor.py:161} INFO - Started process (PID=4210) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:03:24.769-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:03:24.771-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:03:24.770-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:03:24.786-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:03:24.785-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:03:24.787-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:03:24.859-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.101 seconds
[2024-05-09T22:04:52.797-0300] {processor.py:161} INFO - Started process (PID=5089) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:04:52.801-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:04:52.803-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:04:52.802-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:04:52.833-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:04:52.832-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:04:52.834-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:04:52.920-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.138 seconds
[2024-05-09T22:05:31.096-0300] {processor.py:161} INFO - Started process (PID=5150) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:05:31.099-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:05:31.102-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:05:31.101-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:05:31.113-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:05:31.112-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:05:31.114-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:05:31.157-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.070 seconds
[2024-05-09T22:06:07.739-0300] {processor.py:161} INFO - Started process (PID=5208) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:06:07.744-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:06:07.751-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:06:07.751-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:06:07.805-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:06:07.803-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:06:07.806-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:06:07.889-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.167 seconds
[2024-05-09T22:06:42.617-0300] {processor.py:161} INFO - Started process (PID=5268) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:06:42.628-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:06:42.635-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:06:42.635-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:06:42.648-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:06:42.646-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:06:42.656-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:06:42.752-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.143 seconds
[2024-05-09T22:07:29.858-0300] {processor.py:161} INFO - Started process (PID=5332) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:07:30.052-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:07:30.064-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:07:30.053-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:07:30.108-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:07:30.106-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:07:30.109-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:07:32.651-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 2.803 seconds
[2024-05-09T22:08:47.401-0300] {processor.py:161} INFO - Started process (PID=5446) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:08:47.411-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:08:47.416-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:08:47.415-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:08:47.448-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:08:47.447-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:08:47.448-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:08:47.488-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.095 seconds
[2024-05-09T22:09:25.687-0300] {processor.py:161} INFO - Started process (PID=5511) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:09:25.690-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:09:25.692-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:09:25.692-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:09:25.707-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:09:25.705-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:09:25.707-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:09:25.760-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.115 seconds
[2024-05-09T22:10:03.206-0300] {processor.py:161} INFO - Started process (PID=5573) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:10:03.208-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:10:03.211-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:10:03.210-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:10:03.252-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:10:03.249-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:10:03.253-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:10:03.299-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.110 seconds
[2024-05-09T22:10:41.973-0300] {processor.py:161} INFO - Started process (PID=5635) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:10:41.975-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:10:41.977-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:10:41.977-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:10:41.989-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:10:41.988-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:10:41.990-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:10:42.024-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.060 seconds
[2024-05-09T22:11:17.677-0300] {processor.py:161} INFO - Started process (PID=5704) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:11:17.680-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:11:17.685-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:11:17.684-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:11:17.701-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:11:17.699-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:11:17.704-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:11:17.741-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.078 seconds
[2024-05-09T22:11:48.250-0300] {processor.py:161} INFO - Started process (PID=5745) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:11:48.252-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:11:48.256-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:11:48.255-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:11:48.275-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:11:48.273-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:11:48.276-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:11:48.321-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.085 seconds
[2024-05-09T22:12:19.995-0300] {processor.py:161} INFO - Started process (PID=5794) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:12:19.999-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:12:20.007-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:12:20.006-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:12:20.018-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:12:20.017-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:12:20.019-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:12:20.046-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.058 seconds
[2024-05-09T22:12:52.494-0300] {processor.py:161} INFO - Started process (PID=5846) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:12:52.495-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:12:52.497-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:12:52.497-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:12:52.509-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:12:52.508-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:12:52.510-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:12:52.541-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.064 seconds
[2024-05-09T22:13:23.279-0300] {processor.py:161} INFO - Started process (PID=5901) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:13:23.283-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:13:23.290-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:13:23.290-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:13:23.305-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:13:23.304-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:13:23.306-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:13:23.336-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.066 seconds
[2024-05-09T22:13:55.433-0300] {processor.py:161} INFO - Started process (PID=5947) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:13:55.435-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:13:55.437-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:13:55.436-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:13:55.450-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:13:55.449-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:13:55.451-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:13:55.481-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.056 seconds
[2024-05-09T22:14:28.504-0300] {processor.py:161} INFO - Started process (PID=6007) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:14:28.505-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:14:28.507-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:14:28.507-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:14:28.519-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:14:28.518-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:14:28.519-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:14:28.548-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.052 seconds
[2024-05-09T22:14:59.104-0300] {processor.py:161} INFO - Started process (PID=6055) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:14:59.106-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-09T22:14:59.108-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:14:59.107-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:14:59.120-0300] {logging_mixin.py:188} INFO - [2024-05-09T22:14:59.119-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-09T22:14:59.121-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-09T22:14:59.153-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.063 seconds
[2024-05-10T14:17:11.836-0300] {processor.py:161} INFO - Started process (PID=2981) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:17:11.846-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:17:11.855-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:17:11.854-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:17:11.891-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:17:11.888-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:17:11.894-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:17:11.969-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.167 seconds
[2024-05-10T14:18:25.931-0300] {processor.py:161} INFO - Started process (PID=3837) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:18:25.934-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:18:25.937-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:18:25.936-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:18:25.947-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:18:25.946-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:18:25.948-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:18:25.979-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.054 seconds
[2024-05-10T14:19:05.064-0300] {processor.py:161} INFO - Started process (PID=3914) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:19:05.066-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:19:05.069-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:19:05.068-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:19:05.081-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:19:05.080-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:19:05.082-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:19:05.113-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.059 seconds
[2024-05-10T14:19:37.569-0300] {processor.py:161} INFO - Started process (PID=3965) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:19:37.570-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:19:37.572-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:19:37.572-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:19:37.582-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:19:37.581-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:19:37.584-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:19:37.611-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.049 seconds
[2024-05-10T14:20:08.385-0300] {processor.py:161} INFO - Started process (PID=4014) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:20:08.387-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:20:08.392-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:20:08.391-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:20:08.402-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:20:08.401-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:20:08.402-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:20:08.433-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.054 seconds
[2024-05-10T14:20:42.434-0300] {processor.py:161} INFO - Started process (PID=4061) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:20:42.437-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:20:42.439-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:20:42.439-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:20:42.449-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:20:42.448-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:20:42.450-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:20:42.478-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.050 seconds
[2024-05-10T14:21:21.669-0300] {processor.py:161} INFO - Started process (PID=4122) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:21:21.759-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:21:21.761-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:21:21.761-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:21:21.772-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:21:21.771-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:21:21.772-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:21:22.460-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.798 seconds
[2024-05-10T14:21:59.412-0300] {processor.py:161} INFO - Started process (PID=4184) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:21:59.414-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:21:59.417-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:21:59.416-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:21:59.430-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:21:59.428-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:21:59.430-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:21:59.460-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.056 seconds
[2024-05-10T14:22:54.264-0300] {processor.py:161} INFO - Started process (PID=4282) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:22:54.277-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:22:54.279-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:22:54.279-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:22:54.291-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:22:54.290-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:22:54.292-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:22:54.374-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.125 seconds
[2024-05-10T14:23:28.861-0300] {processor.py:161} INFO - Started process (PID=4848) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:23:28.873-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:23:28.875-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:23:28.875-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:23:28.962-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:23:28.908-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:23:28.962-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:23:29.222-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.416 seconds
[2024-05-10T14:24:12.478-0300] {processor.py:161} INFO - Started process (PID=5211) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:24:12.482-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:24:12.484-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:24:12.483-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:24:12.498-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:24:12.493-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:24:12.499-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:24:12.527-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.058 seconds
[2024-05-10T14:30:49.201-0300] {processor.py:161} INFO - Started process (PID=6942) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:30:49.223-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:30:49.236-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:30:49.233-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:30:49.303-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:30:49.294-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:30:49.340-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:30:49.694-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.525 seconds
[2024-05-10T14:32:03.510-0300] {processor.py:161} INFO - Started process (PID=7802) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:32:03.514-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:32:03.517-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:32:03.517-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:32:03.543-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:32:03.541-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:32:03.552-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:32:03.789-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.288 seconds
[2024-05-10T14:32:57.105-0300] {processor.py:161} INFO - Started process (PID=7888) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:32:57.107-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:32:57.110-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:32:57.109-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:32:57.120-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:32:57.119-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:32:57.120-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:32:57.153-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.055 seconds
[2024-05-10T14:33:30.889-0300] {processor.py:161} INFO - Started process (PID=7940) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:33:30.918-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T14:33:30.923-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:33:30.922-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:33:30.952-0300] {logging_mixin.py:188} INFO - [2024-05-10T14:33:30.950-0300] {dagbag.py:355} ERROR - Failed to import: /home/patricia/Documentos/airflow/dags/dados_climaticos.py
Traceback (most recent call last):
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/patricia/Documentos/airflow/dags/dados_climaticos.py", line 7, in <module>
    import pandas as pd
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas.compat import (
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/__init__.py", line 26, in <module>
    from pandas.compat.numpy import is_numpy_dev
  File "/home/patricia/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/pandas/compat/numpy/__init__.py", line 9, in <module>
    _np_version = np.__version__
AttributeError: module 'numpy' has no attribute '__version__'
[2024-05-10T14:33:30.956-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T14:33:31.017-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.146 seconds
[2024-05-10T16:33:05.595-0300] {processor.py:161} INFO - Started process (PID=11588) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:33:05.603-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:33:05.613-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:33:05.611-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:33:07.363-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:33:07.379-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:33:07.378-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:33:07.413-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:33:07.413-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:33:07.484-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.925 seconds
[2024-05-10T16:34:16.981-0300] {processor.py:161} INFO - Started process (PID=11760) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:34:16.984-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:34:16.986-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:34:16.986-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:34:17.501-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:34:17.597-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:34:17.596-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:34:17.613-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:34:17.613-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:34:17.643-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.669 seconds
[2024-05-10T16:34:54.020-0300] {processor.py:161} INFO - Started process (PID=11841) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:34:54.022-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:34:54.028-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:34:54.027-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:34:54.982-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:34:55.011-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:34:55.011-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:34:55.049-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:34:55.049-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:34:55.534-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.522 seconds
[2024-05-10T16:35:34.219-0300] {processor.py:161} INFO - Started process (PID=11912) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:35:34.221-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:35:34.225-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:35:34.224-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:35:34.766-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:35:34.776-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:35:34.776-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:35:34.808-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:35:34.808-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:35:34.840-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.629 seconds
[2024-05-10T16:36:13.184-0300] {processor.py:161} INFO - Started process (PID=12003) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:36:13.189-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:36:13.193-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:36:13.192-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:36:13.697-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:36:13.707-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:36:13.707-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:36:13.724-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:36:13.724-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:36:13.756-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.583 seconds
[2024-05-10T16:36:45.249-0300] {processor.py:161} INFO - Started process (PID=12071) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:36:45.251-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:36:45.254-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:36:45.253-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:36:45.740-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:36:45.755-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:36:45.754-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:36:45.790-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:36:45.789-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:36:45.838-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.596 seconds
[2024-05-10T16:37:18.549-0300] {processor.py:161} INFO - Started process (PID=12139) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:37:18.553-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:37:18.560-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:37:18.559-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:37:19.486-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:37:19.502-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:37:19.501-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:37:19.556-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:37:19.556-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:37:19.654-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.118 seconds
[2024-05-10T16:37:56.008-0300] {processor.py:161} INFO - Started process (PID=12232) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:37:56.017-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:37:56.026-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:37:56.025-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:37:56.740-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:37:56.805-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:37:56.804-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:37:56.878-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:37:56.877-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:37:56.997-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.000 seconds
[2024-05-10T16:38:29.679-0300] {processor.py:161} INFO - Started process (PID=12378) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:38:29.682-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:38:29.685-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:38:29.684-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:38:30.124-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:38:30.174-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:38:30.173-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:38:30.223-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:38:30.222-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:38:30.271-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.599 seconds
[2024-05-10T16:39:29.016-0300] {processor.py:161} INFO - Started process (PID=12524) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:39:29.019-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:39:29.021-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:39:29.021-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:39:29.447-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:39:29.463-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:39:29.463-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:39:29.486-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:39:29.485-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:39:29.516-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.508 seconds
[2024-05-10T16:40:02.341-0300] {processor.py:161} INFO - Started process (PID=12592) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:40:02.343-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:40:02.345-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:40:02.345-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:40:02.888-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:40:02.904-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:40:02.903-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:40:02.933-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:40:02.932-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:40:02.970-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.636 seconds
[2024-05-10T16:40:48.119-0300] {processor.py:161} INFO - Started process (PID=12691) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:40:48.121-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:40:48.123-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:40:48.122-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:40:49.180-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:40:49.190-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:40:49.190-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:40:49.224-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:40:49.224-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:40:49.360-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.247 seconds
[2024-05-10T16:41:55.091-0300] {processor.py:161} INFO - Started process (PID=12841) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:41:55.094-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:41:55.096-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:41:55.096-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:41:55.581-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:41:55.661-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:41:55.661-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:41:55.690-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:41:55.690-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:41:55.722-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.637 seconds
[2024-05-10T16:42:30.513-0300] {processor.py:161} INFO - Started process (PID=12903) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:42:30.516-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:42:30.519-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:42:30.518-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:42:31.014-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:42:31.024-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:42:31.024-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:42:31.043-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:42:31.043-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:42:31.100-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.593 seconds
[2024-05-10T16:43:30.475-0300] {processor.py:161} INFO - Started process (PID=13032) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:43:31.128-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:43:31.131-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:43:31.130-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:43:31.605-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:43:31.615-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:43:31.614-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:43:31.630-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:43:31.630-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:43:32.054-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.586 seconds
[2024-05-10T16:44:37.341-0300] {processor.py:161} INFO - Started process (PID=13128) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:44:37.344-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:44:37.346-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:44:37.346-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:44:37.731-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:44:37.747-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:44:37.747-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:44:37.781-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:44:37.780-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:44:37.814-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.480 seconds
[2024-05-10T16:45:13.161-0300] {processor.py:161} INFO - Started process (PID=13207) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:45:13.163-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:45:13.167-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:45:13.166-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:45:13.609-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:45:13.624-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:45:13.623-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:45:13.645-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:45:13.645-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:45:13.679-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.527 seconds
[2024-05-10T16:46:12.053-0300] {processor.py:161} INFO - Started process (PID=13334) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:46:12.057-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:46:12.062-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:46:12.061-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:46:12.722-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:46:12.736-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:46:12.736-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:46:12.757-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:46:12.757-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:46:12.790-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.752 seconds
[2024-05-10T16:46:42.875-0300] {processor.py:161} INFO - Started process (PID=13418) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:46:42.876-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:46:42.878-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:46:42.878-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:46:43.311-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:46:43.326-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:46:43.325-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:46:43.347-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:46:43.347-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:46:43.378-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.512 seconds
[2024-05-10T16:47:14.890-0300] {processor.py:161} INFO - Started process (PID=13487) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:47:14.895-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:47:14.900-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:47:14.899-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:47:15.566-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:47:15.592-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:47:15.591-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:47:15.630-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:47:15.629-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:47:15.675-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.804 seconds
[2024-05-10T16:47:47.385-0300] {processor.py:161} INFO - Started process (PID=13555) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:47:47.392-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:47:47.397-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:47:47.395-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:47:48.026-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:47:48.044-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:47:48.044-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:47:48.086-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:47:48.086-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:47:48.144-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.801 seconds
[2024-05-10T16:48:45.033-0300] {processor.py:161} INFO - Started process (PID=13688) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:48:45.035-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:48:45.038-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:48:45.038-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:48:45.478-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:48:45.491-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:48:45.491-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:48:45.511-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:48:45.511-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:48:45.547-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.525 seconds
[2024-05-10T16:49:40.294-0300] {processor.py:161} INFO - Started process (PID=13807) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:49:40.296-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:49:40.299-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:49:40.298-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:49:40.715-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:49:40.730-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:49:40.729-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:49:40.749-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:49:40.749-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:49:40.778-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.492 seconds
[2024-05-10T16:50:32.983-0300] {processor.py:161} INFO - Started process (PID=13917) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:50:32.985-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:50:32.988-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:50:32.987-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:50:33.406-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:50:33.421-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:50:33.420-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:50:33.440-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:50:33.440-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:50:33.471-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.496 seconds
[2024-05-10T16:51:18.554-0300] {processor.py:161} INFO - Started process (PID=14011) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:51:18.556-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:51:18.559-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:51:18.558-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:51:18.979-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:51:18.993-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:51:18.992-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:51:19.013-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:51:19.013-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:51:19.044-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.497 seconds
[2024-05-10T16:52:20.117-0300] {processor.py:161} INFO - Started process (PID=14227) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:52:20.125-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:52:20.127-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:52:20.127-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:52:21.463-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:52:28.786-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:52:28.786-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:52:28.843-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:52:28.843-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:52:29.989-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 9.912 seconds
[2024-05-10T16:53:07.988-0300] {processor.py:161} INFO - Started process (PID=14315) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:53:07.991-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:53:07.995-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:53:07.994-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:53:08.638-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:53:08.658-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:53:08.657-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:53:08.705-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:53:08.705-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:53:08.765-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.787 seconds
[2024-05-10T16:53:39.699-0300] {processor.py:161} INFO - Started process (PID=14389) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:53:39.701-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:53:39.704-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:53:39.703-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:53:40.282-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:53:40.299-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:53:40.299-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:53:40.324-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:53:40.324-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:53:40.371-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.679 seconds
[2024-05-10T16:54:35.260-0300] {processor.py:161} INFO - Started process (PID=14504) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:54:35.264-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:54:35.271-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:54:35.270-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:54:35.666-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:54:35.681-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:54:35.681-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:54:35.701-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:54:35.700-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:54:35.737-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.484 seconds
[2024-05-10T16:55:25.597-0300] {processor.py:161} INFO - Started process (PID=14606) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:55:25.602-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:55:25.605-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:55:25.604-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:55:26.130-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:55:26.154-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:55:26.154-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:55:26.182-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:55:26.182-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:55:26.222-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.634 seconds
[2024-05-10T16:55:58.270-0300] {processor.py:161} INFO - Started process (PID=14677) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:55:58.271-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:55:58.274-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:55:58.273-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:55:58.729-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:55:58.745-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:55:58.745-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:55:58.766-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:55:58.766-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:55:58.800-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.539 seconds
[2024-05-10T16:56:54.662-0300] {processor.py:161} INFO - Started process (PID=14882) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:56:54.665-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:56:54.667-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:56:54.666-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:56:55.093-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:56:55.114-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:56:55.113-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:56:55.152-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:56:55.151-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:56:55.192-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.537 seconds
[2024-05-10T16:57:55.382-0300] {processor.py:161} INFO - Started process (PID=15055) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:57:55.397-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:57:55.419-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:57:55.411-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:57:56.009-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:57:56.075-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:57:56.074-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:57:56.130-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:57:56.129-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:57:56.211-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.839 seconds
[2024-05-10T16:58:27.286-0300] {processor.py:161} INFO - Started process (PID=15134) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:58:27.288-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:58:27.290-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:58:27.289-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:58:27.831-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:58:27.846-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:58:27.846-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:58:27.865-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:58:27.864-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:58:28.123-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.859 seconds
[2024-05-10T16:59:07.411-0300] {processor.py:161} INFO - Started process (PID=15213) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:59:07.417-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:59:07.425-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:59:07.423-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:59:08.324-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:59:08.354-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:59:08.353-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:59:08.400-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:59:08.399-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:59:08.480-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.117 seconds
[2024-05-10T16:59:48.189-0300] {processor.py:161} INFO - Started process (PID=15301) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:59:48.194-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T16:59:48.199-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:59:48.198-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:59:49.016-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T16:59:49.031-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:59:49.030-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T16:59:49.050-0300] {logging_mixin.py:188} INFO - [2024-05-10T16:59:49.050-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T16:59:49.085-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.913 seconds
[2024-05-10T17:00:43.039-0300] {processor.py:161} INFO - Started process (PID=15429) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:00:43.041-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:00:43.044-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:00:43.043-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:00:43.586-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:00:43.601-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:00:43.600-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:00:43.621-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:00:43.621-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:00:43.655-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.623 seconds
[2024-05-10T17:01:39.069-0300] {processor.py:161} INFO - Started process (PID=15556) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:01:39.072-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:01:39.075-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:01:39.074-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:01:40.047-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:01:40.070-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:01:40.070-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:01:40.097-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:01:40.096-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:01:40.143-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.082 seconds
[2024-05-10T17:02:38.133-0300] {processor.py:161} INFO - Started process (PID=15660) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:02:38.141-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:02:38.143-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:02:38.143-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:02:38.552-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:02:38.571-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:02:38.570-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:02:38.729-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:02:38.729-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:02:38.766-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.654 seconds
[2024-05-10T17:03:14.414-0300] {processor.py:161} INFO - Started process (PID=15773) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:03:14.424-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:03:14.459-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:03:14.458-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:03:15.555-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:03:15.596-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:03:15.595-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:03:15.667-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:03:15.665-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:03:15.772-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.387 seconds
[2024-05-10T17:04:05.379-0300] {processor.py:161} INFO - Started process (PID=15890) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:04:05.393-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:04:05.398-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:04:05.395-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:04:06.328-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:04:06.357-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:04:06.357-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:04:06.770-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:04:06.770-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:04:06.914-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.552 seconds
[2024-05-10T17:05:45.116-0300] {processor.py:161} INFO - Started process (PID=16022) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:05:45.123-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:05:45.125-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:05:45.124-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:05:45.826-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:05:45.861-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:05:45.860-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:05:45.892-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:05:45.891-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:05:45.960-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.854 seconds
[2024-05-10T17:06:40.874-0300] {processor.py:161} INFO - Started process (PID=16461) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:06:40.877-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:06:40.879-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:06:40.878-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:06:41.387-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:06:41.531-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:06:41.531-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:06:41.561-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:06:41.561-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:06:41.632-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.779 seconds
[2024-05-10T17:07:16.159-0300] {processor.py:161} INFO - Started process (PID=16563) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:07:16.162-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:07:16.170-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:07:16.170-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:07:16.871-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:07:16.890-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:07:16.890-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:07:16.929-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:07:16.929-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:07:16.987-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.836 seconds
[2024-05-10T17:07:52.026-0300] {processor.py:161} INFO - Started process (PID=16633) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:07:52.029-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:07:52.031-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:07:52.030-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:07:52.591-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:07:53.318-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:07:53.317-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:07:53.354-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:07:53.353-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:07:53.403-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.420 seconds
[2024-05-10T17:08:26.142-0300] {processor.py:161} INFO - Started process (PID=16714) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:08:26.144-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:08:26.145-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:08:26.145-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:08:26.706-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:08:26.716-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:08:26.716-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:08:26.733-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:08:26.732-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:08:26.763-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.630 seconds
[2024-05-10T17:08:58.864-0300] {processor.py:161} INFO - Started process (PID=16819) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:08:58.865-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:08:58.867-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:08:58.866-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:09:01.392-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:09:01.405-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:09:01.405-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:09:01.434-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:09:01.433-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:09:01.479-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 2.621 seconds
[2024-05-10T17:09:52.149-0300] {processor.py:161} INFO - Started process (PID=17061) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:09:52.155-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:09:52.158-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:09:52.157-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:09:52.982-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:09:53.003-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:09:53.003-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:09:53.025-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:09:53.025-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:09:53.250-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.110 seconds
[2024-05-10T17:10:37.645-0300] {processor.py:161} INFO - Started process (PID=17143) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:10:37.648-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:10:37.650-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:10:37.649-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:10:38.140-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:10:38.154-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:10:38.154-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:10:38.179-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:10:38.178-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:10:38.216-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.578 seconds
[2024-05-10T17:11:30.117-0300] {processor.py:161} INFO - Started process (PID=17228) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:11:30.121-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:11:30.124-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:11:30.123-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:11:30.668-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:11:30.679-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:11:30.678-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:11:30.697-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:11:30.697-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:11:30.726-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.625 seconds
[2024-05-10T17:12:26.393-0300] {processor.py:161} INFO - Started process (PID=17326) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:12:26.396-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:12:26.398-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:12:26.397-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:12:26.910-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:12:26.920-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:12:26.920-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:12:26.935-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:12:26.935-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:12:26.965-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.578 seconds
[2024-05-10T17:13:16.581-0300] {processor.py:161} INFO - Started process (PID=17407) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:13:16.587-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:13:16.593-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:13:16.589-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:13:17.475-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:13:17.490-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:13:17.490-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:13:17.511-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:13:17.510-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:13:17.549-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.991 seconds
[2024-05-10T17:14:11.647-0300] {processor.py:161} INFO - Started process (PID=17499) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:14:11.650-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:14:11.652-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:14:11.651-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:14:12.202-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:14:12.213-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:14:12.213-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:14:12.232-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:14:12.232-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:14:12.266-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.627 seconds
[2024-05-10T17:15:02.539-0300] {processor.py:161} INFO - Started process (PID=17614) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:15:02.541-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:15:02.543-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:15:02.542-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:15:03.069-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:15:03.081-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:15:03.080-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:15:03.100-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:15:03.100-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:15:03.142-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.611 seconds
[2024-05-10T17:15:57.873-0300] {processor.py:161} INFO - Started process (PID=17758) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:15:57.876-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:15:57.877-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:15:57.877-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:15:58.413-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:15:58.424-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:15:58.424-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:15:58.442-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:15:58.442-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:15:58.474-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.608 seconds
[2024-05-10T17:16:52.858-0300] {processor.py:161} INFO - Started process (PID=17846) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:16:52.860-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:16:52.862-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:16:52.861-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:16:53.395-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:16:53.405-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:16:53.405-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:16:53.422-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:16:53.422-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:16:53.454-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.603 seconds
[2024-05-10T17:17:52.950-0300] {processor.py:161} INFO - Started process (PID=17950) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:17:52.954-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:17:52.962-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:17:52.960-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:17:53.963-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:17:53.982-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:17:53.981-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:17:54.014-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:17:54.014-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:17:54.071-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.149 seconds
[2024-05-10T17:18:25.291-0300] {processor.py:161} INFO - Started process (PID=18002) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:18:25.293-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:18:25.294-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:18:25.293-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:18:25.765-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:18:25.776-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:18:25.775-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:18:25.794-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:18:25.793-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:18:25.822-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.538 seconds
[2024-05-10T17:19:28.505-0300] {processor.py:161} INFO - Started process (PID=18068) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:19:28.511-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:19:28.515-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:19:28.514-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:19:29.245-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:19:29.257-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:19:29.257-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:19:29.274-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:19:29.274-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:19:29.307-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.822 seconds
[2024-05-10T17:20:09.962-0300] {processor.py:161} INFO - Started process (PID=18140) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:20:09.964-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:20:09.965-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:20:09.965-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:20:10.448-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:20:10.457-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:20:10.457-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:20:10.474-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:20:10.474-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:20:10.506-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.550 seconds
[2024-05-10T17:20:58.922-0300] {processor.py:161} INFO - Started process (PID=18227) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:20:58.924-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:20:58.925-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:20:58.925-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:20:59.440-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:20:59.451-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:20:59.450-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:20:59.467-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:20:59.467-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:20:59.496-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.582 seconds
[2024-05-10T17:21:52.457-0300] {processor.py:161} INFO - Started process (PID=18323) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:21:52.460-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:21:52.461-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:21:52.461-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:21:53.001-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:21:53.011-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:21:53.011-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:21:53.030-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:21:53.029-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:21:53.058-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.608 seconds
[2024-05-10T17:22:48.011-0300] {processor.py:161} INFO - Started process (PID=18425) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:22:48.014-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:22:48.016-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:22:48.015-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:22:48.560-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:22:48.570-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:22:48.570-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:22:48.588-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:22:48.588-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:22:48.620-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.617 seconds
[2024-05-10T17:23:40.810-0300] {processor.py:161} INFO - Started process (PID=18537) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:23:40.812-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:23:40.814-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:23:40.813-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:23:41.340-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:23:41.351-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:23:41.350-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:23:41.370-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:23:41.370-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:23:41.399-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.597 seconds
[2024-05-10T17:24:33.384-0300] {processor.py:161} INFO - Started process (PID=18622) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:24:33.387-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:24:33.388-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:24:33.387-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:24:33.890-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:24:33.901-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:24:33.901-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:24:33.927-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:24:33.927-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:24:33.956-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.579 seconds
[2024-05-10T17:25:29.483-0300] {processor.py:161} INFO - Started process (PID=18716) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:25:29.491-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:25:29.493-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:25:29.492-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:25:30.174-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:25:30.191-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:25:30.191-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:25:30.217-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:25:30.216-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:25:30.256-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.781 seconds
[2024-05-10T17:26:24.890-0300] {processor.py:161} INFO - Started process (PID=18811) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:26:24.893-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:26:24.894-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:26:24.894-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:26:25.405-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:26:25.415-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:26:25.415-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:26:25.434-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:26:25.434-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:26:25.470-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.587 seconds
[2024-05-10T17:27:16.914-0300] {processor.py:161} INFO - Started process (PID=18894) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:27:16.917-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:27:16.919-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:27:16.918-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:27:17.420-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:27:17.430-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:27:17.430-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:27:17.446-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:27:17.446-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:27:17.477-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.569 seconds
[2024-05-10T17:28:09.157-0300] {processor.py:161} INFO - Started process (PID=18982) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:28:09.159-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:28:09.161-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:28:09.160-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:28:09.645-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:28:09.668-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:28:09.667-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:28:09.684-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:28:09.684-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:28:09.715-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.565 seconds
[2024-05-10T17:28:42.390-0300] {processor.py:161} INFO - Started process (PID=19042) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:28:42.393-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:28:42.394-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:28:42.394-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:28:42.889-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:28:42.899-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:28:42.899-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:28:42.916-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:28:42.916-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:28:42.950-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.567 seconds
[2024-05-10T17:29:17.040-0300] {processor.py:161} INFO - Started process (PID=19099) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:29:17.042-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:29:17.043-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:29:17.043-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:29:17.555-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:29:17.565-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:29:17.565-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:29:17.582-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:29:17.582-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:29:17.610-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.577 seconds
[2024-05-10T17:30:10.029-0300] {processor.py:161} INFO - Started process (PID=19192) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:30:10.035-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:30:10.036-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:30:10.036-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:30:10.545-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:30:10.555-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:30:10.555-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:30:10.572-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:30:10.572-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:30:10.604-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.582 seconds
[2024-05-10T17:31:00.018-0300] {processor.py:161} INFO - Started process (PID=19278) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:31:00.020-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:31:00.022-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:31:00.021-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:31:00.538-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:31:00.548-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:31:00.547-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:31:00.565-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:31:00.564-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:31:00.594-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.585 seconds
[2024-05-10T17:31:51.954-0300] {processor.py:161} INFO - Started process (PID=19364) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:31:51.957-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:31:51.959-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:31:51.958-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:31:52.629-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:31:52.644-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:31:52.643-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:31:52.669-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:31:52.668-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:31:52.706-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.762 seconds
[2024-05-10T17:32:45.805-0300] {processor.py:161} INFO - Started process (PID=19451) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:32:45.809-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:32:45.812-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:32:45.811-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:32:46.825-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:32:46.847-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:32:46.847-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:32:46.890-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:32:46.890-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:32:46.951-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 1.161 seconds
[2024-05-10T17:33:45.535-0300] {processor.py:161} INFO - Started process (PID=19553) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:33:45.538-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:33:45.541-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:33:45.540-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:33:46.212-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:33:46.227-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:33:46.226-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:33:46.250-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:33:46.250-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:33:46.293-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.768 seconds
[2024-05-10T17:34:47.619-0300] {processor.py:161} INFO - Started process (PID=19649) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:34:47.622-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:34:47.624-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:34:47.623-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:34:48.205-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:34:48.217-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:34:48.217-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:34:48.237-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:34:48.237-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:34:48.269-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.661 seconds
[2024-05-10T17:35:45.983-0300] {processor.py:161} INFO - Started process (PID=19757) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:35:45.988-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:35:45.990-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:35:45.989-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:35:46.641-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:35:46.656-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:35:46.655-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:35:46.679-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:35:46.679-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:35:46.720-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.750 seconds
[2024-05-10T17:36:41.963-0300] {processor.py:161} INFO - Started process (PID=19851) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:36:41.965-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:36:41.966-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:36:41.966-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:36:42.525-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:36:42.536-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:36:42.536-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:36:42.559-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:36:42.558-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:36:42.592-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.638 seconds
[2024-05-10T17:37:37.126-0300] {processor.py:161} INFO - Started process (PID=19941) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:37:37.129-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:37:37.131-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:37:37.130-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:37:37.776-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:37:37.790-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:37:37.790-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:37:37.813-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:37:37.812-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:37:37.855-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.737 seconds
[2024-05-10T17:39:25.531-0300] {processor.py:161} INFO - Started process (PID=20057) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:39:25.533-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:39:25.535-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:39:25.534-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:39:26.167-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:39:26.181-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:39:26.180-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:39:26.206-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:39:26.205-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:39:26.246-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.728 seconds
[2024-05-10T17:40:00.640-0300] {processor.py:161} INFO - Started process (PID=20117) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:40:00.642-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:40:00.643-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:40:00.642-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:40:01.179-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:40:01.189-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:40:01.189-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:40:01.206-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:40:01.206-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:40:01.245-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.612 seconds
[2024-05-10T17:40:54.715-0300] {processor.py:161} INFO - Started process (PID=20203) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:40:54.717-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:40:54.719-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:40:54.718-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:40:55.232-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:40:55.243-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:40:55.242-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:40:55.261-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:40:55.260-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:40:55.290-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.582 seconds
[2024-05-10T17:41:46.070-0300] {processor.py:161} INFO - Started process (PID=20293) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:41:46.073-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:41:46.074-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:41:46.073-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:41:46.619-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:41:46.629-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:41:46.628-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:41:46.646-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:41:46.646-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:41:46.673-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.610 seconds
[2024-05-10T17:42:19.559-0300] {processor.py:161} INFO - Started process (PID=20351) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:42:19.567-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:42:19.568-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:42:19.568-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:42:20.072-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:42:20.082-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:42:20.082-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:42:20.099-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:42:20.099-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:42:20.318-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.766 seconds
[2024-05-10T17:42:51.065-0300] {processor.py:161} INFO - Started process (PID=20409) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:42:51.066-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:42:51.067-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:42:51.067-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:42:51.610-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:42:51.622-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:42:51.622-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:42:51.643-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:42:51.643-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:42:51.677-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.620 seconds
[2024-05-10T17:43:40.935-0300] {processor.py:161} INFO - Started process (PID=20489) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:43:40.937-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:43:40.939-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:43:40.938-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:43:41.568-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:43:41.588-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:43:41.588-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:43:41.611-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:43:41.611-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:43:41.647-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.724 seconds
[2024-05-10T17:44:36.810-0300] {processor.py:161} INFO - Started process (PID=20581) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:44:36.812-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:44:36.813-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:44:36.812-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:44:37.337-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:44:37.348-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:44:37.348-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:44:37.367-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:44:37.366-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:44:37.398-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.596 seconds
[2024-05-10T17:45:30.797-0300] {processor.py:161} INFO - Started process (PID=20674) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:45:30.799-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:45:30.801-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:45:30.800-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:45:31.516-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:45:31.572-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:45:31.572-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:45:31.609-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:45:31.609-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:45:31.666-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.879 seconds
[2024-05-10T17:46:28.010-0300] {processor.py:161} INFO - Started process (PID=20773) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:46:28.013-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:46:28.014-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:46:28.014-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:46:28.536-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:46:28.547-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:46:28.547-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:46:28.567-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:46:28.567-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:46:28.598-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.594 seconds
[2024-05-10T17:47:21.466-0300] {processor.py:161} INFO - Started process (PID=20865) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:47:21.469-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:47:21.470-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:47:21.470-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:47:22.047-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:47:22.057-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:47:22.056-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:47:22.075-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:47:22.075-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:47:22.112-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.653 seconds
[2024-05-10T17:48:19.339-0300] {processor.py:161} INFO - Started process (PID=20962) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:48:19.342-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:48:19.343-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:48:19.343-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:48:19.956-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:48:19.973-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:48:19.973-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:48:19.997-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:48:19.997-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:48:20.041-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.710 seconds
[2024-05-10T17:49:14.334-0300] {processor.py:161} INFO - Started process (PID=21062) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:49:14.337-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:49:14.339-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:49:14.338-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:49:14.952-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:49:14.965-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:49:14.964-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:49:14.985-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:49:14.985-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:49:15.019-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.693 seconds
[2024-05-10T17:50:46.353-0300] {processor.py:161} INFO - Started process (PID=21175) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:50:46.356-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:50:46.358-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:50:46.357-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:50:46.896-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:50:46.908-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:50:46.907-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:50:46.926-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:50:46.925-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:50:46.957-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.612 seconds
[2024-05-10T17:51:29.360-0300] {processor.py:161} INFO - Started process (PID=21321) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:51:29.363-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:51:29.365-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:51:29.364-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:51:30.063-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:51:30.091-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:51:30.091-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:51:30.179-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:51:30.179-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:51:30.234-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.884 seconds
[2024-05-10T17:52:30.913-0300] {processor.py:161} INFO - Started process (PID=21494) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:52:30.916-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:52:30.921-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:52:30.920-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:52:31.439-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:52:31.449-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:52:31.448-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:52:31.466-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:52:31.466-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:52:31.510-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.604 seconds
[2024-05-10T17:53:03.419-0300] {processor.py:161} INFO - Started process (PID=21549) to work on /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:53:03.422-0300] {processor.py:830} INFO - Processing file /home/patricia/Documentos/airflow/dags/dados_climaticos.py for tasks to queue
[2024-05-10T17:53:03.424-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:53:03.423-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:53:04.178-0300] {processor.py:840} INFO - DAG(s) 'dados_climaticos' retrieved from /home/patricia/Documentos/airflow/dags/dados_climaticos.py
[2024-05-10T17:53:04.194-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:53:04.193-0300] {dag.py:3096} INFO - Sync 1 DAGs
[2024-05-10T17:53:04.218-0300] {logging_mixin.py:188} INFO - [2024-05-10T17:53:04.218-0300] {dag.py:3954} INFO - Setting next_dagrun for dados_climaticos to 2024-04-29 00:00:00+00:00, run_after=2024-05-06 00:00:00+00:00
[2024-05-10T17:53:04.257-0300] {processor.py:183} INFO - Processing /home/patricia/Documentos/airflow/dags/dados_climaticos.py took 0.851 seconds
